pl.art.lach.slawek.V2BlankBrowser.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "pl.art.lach.slawek.V2BlankBrowser",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "40",
+  "sdk": "org.gnome.Sdk",
+  "command": "V2BlankBrowser",
+  "clear": [ "*.o"],
+  "build-options" : {
+    "arch": {
+      "x86_64": {
+      },
+      "x86": {
+      }
+    }
+  },
+  "modules": [
+  {
+    "name": "browser-app",
+    "buildsystem" : "cmake-ninja",
+    "sources": [
+    {
+      "type": "git",
+      "url": "https://github.com/nintyfan/V2BlankBrowser.git"
+    }
+    ]
+  }
+  ],
+  
+  "finish-args" : [
+  "--device=dri",
+  "--share=ipc",
+  "--share=network",
+  "--socket=x11",
+  "--socket=pulseaudio"
```

Comment:
you probably want to add `--filesystem=xdg-downloads`

also usually `finish-args` is toward the top of the manifest (as a convention), and the json file properly indented to make it more reviewable.